### PR TITLE
Add pagination tests for monitoring page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Add pagination tests for monitoring page
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | TEST_PATH="functional/BO/03_catalog/03_monitoring/sortAndPagination/*" URL_FO=yourShopURL npm run specific-test
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24215)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js [hunk 1]
```diff
@@ -184,45 +172,65 @@ describe('Sort list of empty categories', async () => {
...
+
+      const paginationNumber = await monitoringPage.selectPaginationLimit(page, tableName, '10');
+      expect(paginationNumber).to.contains(' (page 1 / 2)');
+    });
+
+    it('should click on next', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnNext', baseContext);
+
+      const paginationNumber = await monitoringPage.paginationNext(page, tableName);
+      expect(paginationNumber).to.contains(' (page 2 / 2)');
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js]: Do we really care about this space?

File: tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js [hunk 2]
```diff
@@ -184,45 +172,65 @@ describe('Sort list of empty categories', async () => {
...
-      {args: {categoryToCreate: secondCreateCategoryData}},
-      {args: {categoryToCreate: thirdCreateCategoryData}},
-    ];
+  // 3 : Pagination
+  describe('Pagination next and previous', async () => {
+    it('should change the items number to 10 per page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo10', baseContext);
+
+      const paginationNumber = await monitoringPage.selectPaginationLimit(page, tableName, '10');
+      expect(paginationNumber).to.contains(' (page 1 / 2)');
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js]: Do we really care about this space?

File: tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js]: It's a mistake, I will fix it! thank you

Comment (NeOMakinG) [reply tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js]: Ok np, it contains some more above ;)

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**